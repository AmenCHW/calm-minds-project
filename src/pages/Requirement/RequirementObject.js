export const WhyWorkWithHealingObject = [
    {
        id: 1,
        header: 'RELIABLE INCOME',
        text: 'Over 10,000 people sign up on BetterHelp every day looking for a counselor to help with lifeâ€™s challenges. BetterHelp can be your main source of income ("full time") or a supplement to your current work.',
    },
    {
        id: 2,
        header: 'FOCUS ON COUNSELING',
        text: 'No need to worry about costs from acquiring clients, billing, support or operations. Let us handle the fees and paperwork so you can focus on what you do best!',
    },
    {
        id: 3,
        header: 'FOCUS ON COUNSELING',
        text: 'No need to worry about costs from acquiring clients, billing, support or operations. Let us handle the fees and paperwork so you can focus on what you do best!',
    },
];

export const RequirementsObject = [
    {
        id: 1,
        text: "Licensed by a State Board to provide counseling"
    },
    {
        id: 2,
        text: "Experience in counseling for adults, couples, and/or teens"
    },
    {
        id: 3,
        text: "Excellent writing skills"
    },
    {
        id: 4,
        text: "Reliable Internet connection"
    },
    {
        id: 5,
        text: "Currently residing in the US"
    },
]