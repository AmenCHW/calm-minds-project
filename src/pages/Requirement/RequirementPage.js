import React from 'react';
import img1 from './Frame.png';

const RequiredsObject = [
  {
    id: 1,
    header:'RELIABLE INCOME',
    text: 'Over 10,000 people sign up on BetterHelp every day looking for a counselor to help with lifeâ€™s challenges. BetterHelp can be your main source of income ("full time") or a supplement to your current work.',

  {
    id: 2,
    header:'FOCUS ON COUNSELING',
    text: 'No need to worry about costs from acquiring clients, billing, support or operations. Let us handle the fees and paperwork so you can focus on what you do best!',
  },
  {
    id: 3,
    header:'FOCUS ON COUNSELING',
    text: 'No need to worry about costs from acquiring clients, billing, support or operations. Let us handle the fees and paperwork so you can focus on what you do best!',
  },
];

export default function Requirement() {
  return (
    <div className="flex flex-wrap mx-auto lg:max-w-7xl px-10 py-10 font-[ 'Poppins']">
      <div >
        <h1 className='text-3xl sm:text-4xl md:text-5xl format-normal leading-normal pt-5'> WHY WORK WITH HEALING ?</h1>
        {RequiredsObject.map((element)=>{
          return ( <div className='mt-5'>
            <h3 className='mt-1 text-xs font-bold'>{element.header}</h3>
            <p className=" pr-10 max-w-lg text-[14px] text-gray-500 ">{element.text}</p>
            </div>);  
           })}  
            <div  className=" mt-6 text-xl font-semibold">REQUIRENMENTS</div>
            <ul className='list-disc list-inside text-black'>
            <li className=" mt-2 text-xs  text-gray-500 ">Licensed by a State Board to provide counseling</li>
            <li className=" mt-2 text-xs  text-gray-500 ">Experience in counseling for adults, couples, and/or teens</li>
            <li className=" mt-2 text-xs  text-gray-500 ">Excellent writing skills</li>
            <li className=" mt-2 text-xs  text-gray-500 ">Reliable Internet connection</li>
            <li className=" mt-2 text-xs  text-gray-500 ">Currently residing in the US</li>
            </ul>
            <button
            className="my-4 bg-[#2DD3E3] hover:bg-blue-400 text-white 
                  font-bold py-2 px-4 border-b-4 border-[#2DD3E3] hover:border-blue-500 rounded"
            type="button"
          >
           GET STARTED
          </button>
      </div>
      <div> 
         <img
          className="mt-3 md:mt-0 h-auto lg:h-96 xl:h-auto xl:w-auto"
          src={img1}
          alt="loading"
        />
        </div>
    </div>
  );
}
