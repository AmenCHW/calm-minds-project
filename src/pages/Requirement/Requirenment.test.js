import React from 'react';
// import {screen} from "@testing-library/react"
import Requirement from './RequirementPage';
import renderer from 'react-test-renderer';

test('first snapshot testing', () => {
  const component = renderer.create(<Requirement />).toJSON();
  expect(component).toMatchSnapshot();
});

// test("render 5 requirenments",()=>{
//     const list=screen.getAllByRole("listitem")
//     console.log(list)
//     expect(list).toHaveLength(5);
// })
