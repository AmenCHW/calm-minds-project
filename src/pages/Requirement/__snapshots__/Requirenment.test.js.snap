// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`first snapshot testing 1`] = `
<div
  className="flex flex-wrap mx-auto lg:max-w-7xl px-10 py-10 font-[ 'Poppins']"
>
  <div>
    <h1
      className="text-3xl sm:text-4xl md:text-5xl format-normal leading-normal pt-5"
    >
       
      WHY WORK WITH HEALING ?
    </h1>
    <div
      className="mt-5"
    >
      <h3
        className="mt-1 text-xs font-bold"
      >
        RELIABLE INCOME
      </h3>
      <p
        className=" pr-10 max-w-lg text-[14px] text-gray-500 "
      >
        Over 10,000 people sign up on BetterHelp every day looking for a counselor to help with lifeâ€™s challenges. BetterHelp can be your main source of income ("full time") or a supplement to your current work.
      </p>
    </div>
    <div
      className="mt-5"
    >
      <h3
        className="mt-1 text-xs font-bold"
      >
        FOCUS ON COUNSELING
      </h3>
      <p
        className=" pr-10 max-w-lg text-[14px] text-gray-500 "
      >
        No need to worry about costs from acquiring clients, billing, support or operations. Let us handle the fees and paperwork so you can focus on what you do best!
      </p>
    </div>
    <div
      className="mt-5"
    >
      <h3
        className="mt-1 text-xs font-bold"
      >
        FOCUS ON COUNSELING
      </h3>
      <p
        className=" pr-10 max-w-lg text-[14px] text-gray-500 "
      >
        No need to worry about costs from acquiring clients, billing, support or operations. Let us handle the fees and paperwork so you can focus on what you do best!
      </p>
    </div>
    <div
      className=" mt-6 text-xl font-semibold"
    >
      REQUIRENMENTS
    </div>
    <ul
      className="list-disc list-inside text-black"
    >
      <li
        className=" mt-2 text-xs  text-gray-500 "
      >
        Licensed by a State Board to provide counseling
      </li>
      <li
        className=" mt-2 text-xs  text-gray-500 "
      >
        Experience in counseling for adults, couples, and/or teens
      </li>
      <li
        className=" mt-2 text-xs  text-gray-500 "
      >
        Excellent writing skills
      </li>
      <li
        className=" mt-2 text-xs  text-gray-500 "
      >
        Reliable Internet connection
      </li>
      <li
        className=" mt-2 text-xs  text-gray-500 "
      >
        Currently residing in the US
      </li>
    </ul>
    <button
      className="my-4 bg-[#2DD3E3] hover:bg-[#6ae8f3] text-white  font-bold py-2 px-4 border-b-4 border-[#2DD3E3] hover:border-[#54deeb] rounded"
      type="button"
    >
      GET STARTED
    </button>
  </div>
  <div>
    <img
      alt="loading"
      className="mt-3 md:mt-0 h-auto lg:h-96 xl:h-auto xl:w-auto"
      src="Frame.png"
    />
  </div>
</div>
`;
